mobile/components/PastEntryCard.tsx

```diff
@@ -0,0 +1,53 @@
+import { View, StyleSheet, Text } from 'react-native'
+//@ts-ignore
```

Comment:
Generally, we shouldnt use `@ts-ignore` bc it alters compilation errors. It looks like its commented out though, so deleting it would help with overall code quality